docs/en/observability/monitor-infra/analyze-hosts.asciidoc

```diff
@@ -0,0 +1,170 @@
+<<add-observability-data>>.
+
+[discrete]
+[[analyze-hosts-filter-view]]
+== Filter the Hosts view
+
+The **Hosts** page provides several mechanisms for filtering the data on the
+page:
+
+* Enter a search query to show metrics that match your search criteria. For
+example, to see metrics for all hosts running on Google Cloud Platform (GCP),
+enter `cloud.account.provider : "gcp"`. Otherwise you'll see metrics for all
+your monitored hosts.
+* Select additional criteria to filter the view: 
+** In the **Operating Systems** list, select one or more operating systems
+and include (or exclude) metrics for those hosts.
+** In the **Cloud providers** list, select one or more cloud cloud providers to
+include (or exclude) metrics for the selected cloud providers. 
+* Change the date range in the Time Picker, or click and drag on a
+visualization to change the date range.
+* In the metrics visualizations, click a point on a line and apply filters to
+set the view to the same time and host.
+
+To learn more about filtering data in {kib}, refer to
+{kibana-ref}/kibana-concepts-analysts.html[{kib} concepts].
+
+[NOTE]
+.Limitation on filtering by service names
+====
+When filtering on a specific service name, the **Hosts** page may display hosts,
+but no metrics. This may happen when you are using APM because it inserts the
+service name into documents that contain a hostname, but no metrics.
+====
+
+[discrete]
+[[analyze-hosts-open-in-lens]]
+== Open in Lens
+
+Because all visualizations are powered by Lens, you can continue your analysis
+in Lens if you require more flexibility. Under **Metrics**, hover your cursor
+over a visualization, then click the Ellipses icon in the upper-right corner to
+open the visualization in Lens.
+
+In Lens, you can examine all the fields and formulas used to create the
+visualization, make modifications to the visualization, and save your changes.
+
+For more information about using Lens, refer to the
+{kibana-ref}/lens.html[{kib} documentation about Lens].
+
+[discrete]
+[[analyze-hosts-inspect-data]]
+== Inspect and download metrics
+
+From the **Hosts** page, you can access a text-based view of the data underlying
+your metrics visualizations and optionally download the data to a
+comma-separated (CSV) file. 
+
+Under **Metrics**, hover your cursor over a visualization, then in the
+upper-right corner, click the Ellipses icon to inspect the data.
+
+In the flyout, click **Download CSV** to download formatted or raw data to a CSV
+file. 
+
+Notice that you can change the view to **View: Requests** to explore the request
+used to fetch the data and the response returned from {es}. You can click links
+to further inspect and analyze the request in the **Dev Console** or
+**Search Profiler**.
+
+[discrete]
+[[analyze-hosts-fields]]
+== Field reference
+
+The following sections show the formulas used to populate the **Hosts** page
+with data.
+
+****
+**Question for reviewers**
```

Comment:
@roshan-elastic This doc is not quite ready for review, but you could really help me out by a) indicating whether this info is useful, and b) provide a plain english description of how the value for each metric is calculated.

If you think providing the formula is good enough, then help me figure out the formula used by the tiles at the top of the page. I looked in the code, but it was taking too long so decided it's better to ask.

Reply:
This is fantastic! Thank you!

Reply:
Updated in 40b6ba4c1b4c613363b7e8b4b9106093c9c23cd3.